packages/grafana-data/src/datetime/timezones.test.ts

```diff
@@ -2,28 +2,25 @@ import { getTimeZoneInfo } from './timezones';
 import { setTimeZoneResolver } from './common';
 
 describe('getTimeZoneInfo', () => {
-  // global timezone is set to UTC, see jest-config.js file
+  // global timezone is set to Pacific/Easter, see jest-config.js file
 
   describe('IANA canonical name of the timezone', () => {
     it('should resolve for default timezone', () => {
       setTimeZoneResolver(() => 'browser');
       const result = getTimeZoneInfo('', Date.now());
-      expect(result?.ianaName).toBe('Africa/Abidjan');
+      expect(result?.ianaName).toBe('Pacific/Easter');
```

Comment:
This `Africa/Abidjan` was weird. I assume it's because utc isn't a real timezone so when moment tried to infer the timeozne it jest returned the first +00:00 timezone there is alphabetically.

Reply:
Agree, that looks wieird.